src/libraries/System.Numerics.Tensors/ref/System.Numerics.Tensors.csproj

```
@@ -1,6 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
-    <TargetFrameworks>$(NetCoreAppCurrent);$(NetCoreAppPrevious);$(NetCoreAppMinimum);netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent);netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
```

[Now that `NetCoreAppCurrent` means net9.0](https://github.com/dotnet/runtime/commit/85fa924eab86aedd55dc751c8ef56cf2cc66d31), we should add a net8.0 minimum TFM as well.

```suggestion
    <TargetFrameworks>$(NetCoreAppCurrent);$(NetCoreAppPrevious);$(NetCoreAppMinimum);netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
```